src/OSPSuite.UI/UIConstants.cs

```diff
@@ -69,19 +69,18 @@ private static int computedExtraRowHeight()
 
          public static int ScaleForScreenDPI(int size, int? maxSize = null)
          {
-            var proposedSize = (int) (_scaleFactor * size);
+            var proposedSize = (int)(_scaleFactor * size);
             return Math.Min(proposedSize, maxSize.GetValueOrDefault(proposedSize));
          }
 
          private static double createScaleFactor()
          {
             using (var graphics = Graphics.FromHwnd(IntPtr.Zero))
             {
-               if (graphics.DpiX >= 200)
-                  return 2;
-
                if (graphics.DpiX > 120)
```

Comment:
Maybe we should just always scale if DpiX is > 96. I didn't want to jump the gun though.

Here, my DPI reading was 192 - just below the 200 required to scale by a factor of 2.

I adjusted the threshold to 190 instead of 200 and it looked good.

I guess the DPI will always be the % value of scaling x 96. So it stands to reason we should just scale as % of 96.

Reply:
yeah.. I agree. I would feel it should be always that scaling /96 but cool